### ISSUE TITLE
Status für Genesen-Geimpfte wird falsch angezeigt

### ISSUE THREAD

**Issue by GN81** [corona-warn-app/cwa-wishlist#770]
Leider wird der Status von Genesen und Geimpften nicht richtig in den gängigen Apps (CovPass und Corona Warn App) dargestellt. Ist man z.B. Genesen und danach mit Johnson&Johnson geimpft wird der Impfstatus immer mit 1/1 dargestellt. Und man gilt nun als ungeimpft. 
Apotheken und Ärzte haben keine Möglichkeit dies korrekt auszugeben und man selbst lädt dann falsche Zertifikate in die Apps. 
Selbst J&J Impfung und die nun nötigen 2 Booster können nicht als 3/3 sondern maximal als 2/2 angezeigt werden. 
Leider gibt es so nun immer Diskussionen und man wird überall abgewiesen.

---
Internal Tracking ID: [EXPOSUREAPP-11557](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11557) Booster rule
Internal Tracking ID: [EXPOSUREAPP-11558](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11558) Business rule

**Comment by Ein-Tim** [comment]
Zum 2/2 Booster gibt es gute Nachrichten: https://www.coronawarn.app/de/faq/#vac_booster_jj

**Comment by GN81** [comment]
Hallo. Danke für die Antwort.

_„Die erste Impfung nach einer Genesung von COVID-19 wird mit „1 von 1“ im Zertifikat eingetragen. Erfolgt diese Impfung für Genesene mit dem Impfstoff von J&J kann diese nicht von der Grundimmunisierung mit J&J unterschieden werden, da in beiden Fällen „1 von 1“ im Zertifikat vermerkt wird.“_

Leider kann der normale Bürger dies, wie die App, nicht unterscheiden und man wird als ungeimpft bezeichnet und nicht als 2G.

Wenn dann der Booster drauf kommt gilt man in der App als geimpft (2G), obwohl der korrekte Status 1G+ wäre,  aber man muss immer einen Test bei sich führen, da auch hier der normale Bürger es nur als 2G erkennt und nicht als 2G+. 
Das ist diskriminierend und muss geändert werden. 
Ich möchte betonen das dies auch bei „genesen + geimpft“ mit anderen Impfstoffen so geht. Dieses Problem ist seit 9 Monaten bekannt und wurde nicht geändert.

**Comment by Ein-Tim** [comment]
> Dieses Problem ist seit 9 Monaten bekannt und wurde nicht geändert.

Doch, das wird sich zum 01.02. ändern. Ab dann wird der Booster nämlich nicht mehr als "2 von 2" ausgestellt sondern als "2 von 1" ausgestellt. Steht auch in dem verlinkten FAQ Eintrag.

**Comment by GN81** [comment]
Das ändert ja immernoch nichts daran das die Genesung nirgends erfasst/aufgeführt wird. Folglich wird der Status mit Genesung + Impfung immernoch als ungeimpft und Genesung + Impfung + Booster nur als 2G angezeigt. Die Genesung wird nicht angezeigt.

**Comment by Ein-Tim** [comment]
@GN81 

Man kann sich halt ein Genesenenzertifikat ausstellen lassen aber generell hast du Recht, dass wurde bei der Spezifikation durch die EU vergessen.

**Comment by GN81** [comment]
Die EU hat die Bundesregierung schon lange angewiesen das zu ändern.passiert nur leider nicht. Das Genesenenzertifikat kann man sich nachträglich, hab es damals wegen der Impfung nicht gebraucht, nicht mehr ausstellen lassen. Das wird durch die entsprechende Software blockiert.

**Comment by Ein-Tim** [comment]
@GN81 
Die EU hat Deutschland aufgefordert? Ich wüsste nicht, was Deutschland hier ändern könnte. 

Das 1/1 J&J Booster nicht von 1/1 J&J normale Impfungen unterschieden werden können, ist kein Fehler den Deutschland beheben könnte.

**Comment by GN81** [comment]
Soweit ich weiß hat das RKI die covpass App entwickelt. Wenn eine Genesung + Impfung genauso angezeigt wird wie eine einfache Impfung etc. müsste man die App ändern. Da das RKI dem Gesundheitsministerium unterstellt ist, stellt sich die Frage der Zuständigkeit nicht. Es ist ein Fehler der auf EU Ebene bekannt ist und an die entsprechenden Länderstellen weitergegeben wurde. Es ist generell belanglos wer zuständig ist. Es ist ein Fehler auf Kosten der Bürger der korrigiert werden muss.

**Comment by Ein-Tim** [comment]
@GN81 

Da stimme ich dir zu, dass Problem muss gelöst werden. 

> Soweit ich weiß hat das RKI die covpass App entwickelt. Wenn eine Genesung + Impfung genauso angezeigt wird wie eine einfache Impfung etc. müsste man die App ändern. Da das RKI dem Gesundheitsministerium unterstellt ist, stellt sich die Frage der Zuständigkeit nicht. 

Das ist korrekt. Was aber auch korrekt ist, ist dass eine 1/1 normale J&J Impfung nicht von einem 1/1 Booster unterschieden werden können ist ein _Fehler_ in der EU-Spezifikation, der auf EU-Ebene gelöst werden muss.

**Comment by GN81** [comment]
Wenn das so sein sollte, ist es noch fahrlässiger die Covid-Regelungen auf Bundesebene zu ändern ohne das es eine Übergangsfrist bis zur Lösung des App-Problems gibt. Absolut unprofessionell.

**Comment by vaubaehn** [comment]
@GN81 
Auch ich stimme dir zu, dass da ganz schön viel schief gelaufen ist.
Was die Nummerierung der Zertifikate angeht, die im Konsens der EU entwickelt worden sind, habe ich die Ansicht, dass da einfach auch ein bisschen Pech dabei war. Die Zertifikate wurden unter einem erheblichen Zeitdruck entwickelt, als die Impfungen hier erst gerade anliefen. Zu dem Zeitpunkt war noch gar nicht absehbar, dass einige Impfungen als Einzeldosis ausreichend sein _könnten_, ob zwei Impfdosen für eine lebenslange Immunität reichen, und wie sich Infektionen in Kombination mit Impfungen auf den Immunstatus auswirken. Sprich: im Frühjahr 2021 hat noch niemand wissen können, dass wir mal über 2G/3G plus und minus mit und ohne Booster reden müssen... So gesehen würde ich der EU keinen Vorwurf machen wollen.
Die _Dynamik_ der Lage,  das Auftauchen neuer Varianten etc. macht es erforderlich, politisch schnell zu reagieren, und Gegenbenheiten (wie Zugangsvoraussetzungen) schnell umzusetzen. Leider kommt man bei den schnellen Entschlüssen gar nicht mehr mit dem Programmieren hinterher...
Leider setzt dann aber dem Ganzen die Krone auf, dass Beschlüsse zu Zugangsvoraussetzngen, insbesondere, was wann als Genesen, vollständig geimpft, grundimmunisiert oder geboostert auch noch in einzelnen Bundesländern unterschiedlich gehandhabt wird! Wie soll man das in der App umsetzen?!
Tatsächlich wird es dafür voraussichtlich Mitte Februar eine Lösung geben (und ich hoffe, CCL bzw. Admission Rules können auf Länderebene heruntergebrochen werden, @mlenkeit / @thomasaugsten ?).
Jetzt kann man sagen, das hätte es schon deutlich früher geben müssen. Da stimme ich zu. Ich kann nur mutmaßen, dass der Regierungswechsel im letzten Jahr einiges behindert bzw. verzögert hat... Die Verträge mit SAP/TSI/IBM wurden den Medien nach erst vor kurzem verlängert...
SAP und TSI können da naturgemäß nicht viel machen, also liegt es an uns, uns möglichst direkt ans BMG  zu wenden und die Dringlichkeit der Lage darzustellen.
Für das Thema, über das wir hier gerade reden, ist es zu spät - denn das ist jetzt bereits auf dem Weg der Umsetzung, und braucht nun solange, wie es eben braucht.
Aber für die Zukunft sollten wir ein wachsames Auge haben, um rechtzeitig beim BMG auf der Matte zu stehen. Aus nachvollziehbaren Gründen können uns SAP/TSI/IBM leider nicht einfach verraten, wenn es beim BMG hakt...

**Comment by GN81** [comment]
Vielen Dank für die tolle Antwort. Und genau deine Worte werde ich mir morgen zu Herzen nehmen und beim BMG telefonisch vorstellig werden.

**Comment by vaubaehn** [comment]
@GN81 
Super, vielen Dank! Wenn du jemanden erreichst, es gibt noch ein anderes wichtiges Thema: die Geschäftszeiten/Kapazitäten der TAN-Hotline sollten dringend ausgeweitet werden! https://github.com/corona-warn-app/cwa-hotline/issues/14

**Comment by Ein-Tim** [comment]
@GN81 

Seit 01.02.22 können Apotheken für solche Szenarien ein 2 von 1 bzw. auch ein 3 von 1 Zertifikat erstellen. Hast du dir so ein Zertifikat geben lassen?
